### PR TITLE
Implemen std::fmt::Display for String32

### PR DESCRIPTION
**Pull request by xla**
When handling the String32 type it's sometimes required to get the underlying String it wraps in order hand over to clients. As advises this is done by implementing the Display trait from std::fmt.

***
@geigerzaehler Working of your `consolidate-client-packages` branch and needed `to_string` on `String32`. Let me know if you want this change in a different form and would like to see tests for it.

### CODE REVIEW

**Comment by rockbmb** [comment]
@xla you can merge this now, if you still need it.

File: runtime/src/registry.rs [hunk 1]
```diff
@@ -50,6 +51,12 @@ impl FromStr for String32 {
     }
 }
 
+impl fmt::Display for String32 {
```
Comment (geigerzaehler) [inline runtime/src/registry.rs]: Could you add `#[cfg(feature = "std")]` so that we only compile this for the client?

Comment (xla) [reply runtime/src/registry.rs]: Is that true even when we use the `sr_std` crate?

**Comment by rockbmb** [comment]
Did you mean to close this?

**Comment by xla** [comment]
@rockbmb No, but the base branch vanished and I can't edit it....dumb UI. Gonna open a new one.

**Pull request closed without merging**